perfzero/lib/perfzero/perfzero_config.py

```diff
@@ -45,15 +61,15 @@ def __init__(self, mode):
           'ROGUE_REPORT_PROJECT', False, default='')
       self.bigquery_table_name_str = 'benchmark_results.result'
 
-      if self.bigquery_project_name_str == '':
+      if self.bigquery_project_name_str:
```

Comment:
Should this be `if not self.bigquery_project_name_str `?

Previously when `ROGUE_PEPORT_PROJECT` is not explicitly defined, PerfZero will not report any result to bigquery. If it is defined, PerfZero will report result to table `benchmark_results.result` of the bigquery project `google.com:tensorflow-performance `.

The purpose of this "if" statement is for PerfZero to report result to table `benchmark_results_dev.result` of the bigquery project `google.com:tensorflow-performance ` if `ROGUE_PEPORT_PROJECT` is not explicitly defined.

Reply:
You do not have to write all that out.  In the future just say: "hey that is checking for the negative,  `if not self.bigquery....:`  will fix it.    

lint was complaining about it and I just did not think through it being reverse.

Reply:
will submit update tomorrow morning

Reply:
Sure! I will do it in the future.